wicket/src/ui/defaults/dimensions.rs

```diff
@@ -30,7 +30,7 @@ pub trait RectExt {
 
 impl RectExt for Rect {
     fn center_horizontally(mut self, width: u16) -> Self {
-        let center = (self.width - width) / 2;
+        let center = (self.width.saturating_sub(width)) / 2;
```

Comment:
When shrinking the terminal during dev. It's not critical, just a robustness thing. We should probably be using saturating arithmetic in lots of cases to prevent crashes.